doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -4,6 +4,29 @@
 Inputs and Outputs
 *************************
 
+The ufs-weather-model can be configured as one of several applications, from a single component model atmosphere model to a fully coupled model with multiple earth system components (atmosphere, ocean, sea-ice and mediator). Currently the supported configurations are:
+
+.. _UFS-applications:
+
+.. list-table:: *Supported ufs-weather-model applications*
+   :widths: 10 70
+   :header-rows: 1
+   
+   * - App name
+     - Description
+   * - ATM
+     - Standalone UFSAtm
+   * - ATMW
+     - UFSAtm coupled to WW3
+   * - ATMAERO
+     - UFSAtm coupled to GoCart
+   * - S2S
+     - Coupled UFSATM-MOM6-CICE6-CMEPS
+   * - S2SW
+     - Coupled UFSATM-MOM6-CICE6-WW3-CMEPS
+   * - DATM
```

Comment:
@binli2337 should this be NG_GODAS? Do you also want to have NG-GODAS-NEMSDATM?

Reply:
@junwang-noaa @DeniseWorthen I think DATM should be changed to NG-GODAS. I am not sure whether we need to include NG-GODAS-NEMSDATM and related input files?

Reply:
I've changed the name here to be NG_GODAS. I did not think we would be adding anything for NEMSDATM, only for CDEPS as a DATM.